source/_webhooks.html.md.erb

```diff
@@ -434,6 +436,12 @@ For more information, see [Intro to Webhooks](#intro-to-webhooks).
 **eventType** | string | Type of event that occurred. One of the following values: **created**, **deleted**, **updated**
 **timestamp** | timestamp | Time that this event occurred. A callback may contain events with different timestamps, as multiple separate events may be aggregated into a single callback request.
 
+## ColumnIds Object
```

Comment:
The `columnIds` field isn't really an object that requires documentation, it's just an array of numbers.

In other words, it's not a structured object with a well-defined set of fields/properties that need to be documented.

Reply:
Yeah, I went back and forth on this one. I'm happy to keep it simple.